apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -38,66 +40,81 @@ import {
 } from '../api';
 import { normalizeWriteInName } from '../utils/write_ins';
 
-const BallotViews = styled.div`
-  flex: 3;
-  background: #455a64;
-  padding: 0 0.5rem;
+const AdjudicationScreen = styled(Screen)`
+  /* Matches the focus style applied in libs/ui/global_styles.tsx, which are
+   * disabled by default in VxAdmin and enabled on the touch-only VxSuite
+   * machines.
+   * TODO: We should probably figure out a more consistent approach to
+   * enabling/disabling focus outlines across a single app, instead of
+   * conditionally enabling on certain pages.
```

Comment:
I think there was an intention when this screen was designed to make it super easy to navigate by keyboard so that you could adjudicate a lot of write-ins quickly, hence the need for focus outlines. I don't know whether or not users actually use it. My hunch would be no, so we could probably just drop it.

Reply:
Sounds good! I've noticed we spend a lot of effort maintaining it, so I'm onboard with dropping that requirement